app/modules/ws2812.c

```diff
@@ -165,6 +165,110 @@ static int ws2812_buffer_fade(lua_State* L) {
   return 0;
 }
 
+static int ws2812_buffer_fade_in(lua_State* L) {
```

Comment:
Duplicated code with `ws2812_buffer_fade_out()`. If you unify the API as mentioned below, then there's no need for two separate functions anymore.